server/database_manager/src/main.rs

```diff
@@ -2,60 +2,37 @@
 
 // Std.
 use std::env;
-use std::fs;
 use std::io;
 use std::io::*;
 
 // Custom.
 use shared::misc::db_manager::*;
 
-/// Looks if there is a database file in the current directory.
-/// Returns `true` if a database was found, `false` if not.
-fn find_database() -> bool {
-    let paths = fs::read_dir(".");
-    if let Err(e) = paths {
-        panic!("{}", e);
-    }
-    let paths = paths.unwrap();
-
-    for path in paths {
-        if let Err(e) = path {
-            panic!("{}", e);
-        }
-        let path = path.unwrap();
-
-        if path.file_type().unwrap().is_file()
-            && path.file_name().to_str().unwrap() == DATABASE_NAME
-        {
-            return true;
-        }
-    }
-
-    false
-}
-
 fn main() {
     println!(
         "FBugReporter (database manager) (v{}).",
         env!("CARGO_PKG_VERSION")
     );
     println!("Type 'help' to see commands...\n");
 
-    if !find_database() {
+    let database_location = DatabaseManager::get_database_location();
+
+    println!(
+        "Looking for database in '{}'...",
```

Comment:
Add `INFO:` text in the beginning of the message.

Reply:
Once we migrate to better logging system this will be handled by e.g. `log::info!`.
But changed this for now.